src/lib/components/form/BuildForm.svelte

```diff
@@ -430,7 +430,10 @@
       class="form-select"
       name="tags"
       aria-describedby="tags"
-      bind:value={build.tags}
+      bind:value={
+        () => build.tags.map((tag) => tag.id),
+        (v) => (build.tags = tags.filter((tag) => v.some((id) => id === tag.id)))
+      }
```

Comment:
What's going on here 👀

Reply:
The `tags` field of a fully-fledged build is a
```
{
  id: number,
  label: string
}[]
```
but within the form, we need to treat it as an array of primitive values (their IDs). Svelte actually gives us a really nice way to translate directly in the bind expression by doing this! The first function yields an expression which evaluates to the node's property, and the second function determines how the node's property is translated to the rest of our script.

More information: [Svelte - bind - Function bindings](https://svelte.dev/docs/svelte/bind#Function-bindings)